doc/index.rst

```diff
@@ -1,12 +1,40 @@
-####################################
-ts_ess_sensors documentation preview
-####################################
+.. py:currentmodule:: lsst.ts.ess_sensors
 
-.. This page is for local development only. It isn't published to pipelines.lsst.io.
+.. _lsst.ts.ess_sensors:
 
-.. Link the index pages of package and module documentation directions (listed in manifest.yaml).
+###################
+lsst.ts.ess_sensors
+###################
 
-.. toctree::
-   :maxdepth: 1
+.. Paragraph that describes what this Python module does and links to related modules and frameworks.
```

Comment:
Could you please fill in this information -- at least with basic info?

Reply:
Done